File: src/main/java/com/explosion204/wclookup/model/entity/Ticket.java
package com.explosion204.wclookup.model.entity;

import lombok.Data;

import javax.persistence.*;
import java.time.LocalDateTime;

@Data
@Entity
@Table(name = "ticket")
public class Ticket {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    private String subject;
    private String text;
    private String email;
    private LocalDateTime creationTime;
    private boolean isResolved;
}

File: android/app/src/main/java/com/example/wclookup/core/model/Ticket.kt
package com.example.wclookup.core.model

import com.example.wclookup.core.constant.NameConstant.EMAIL
import com.example.wclookup.core.constant.NameConstant.ID
import com.example.wclookup.core.constant.NameConstant.SUBJECT
import com.example.wclookup.core.constant.NameConstant.TEXT
import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class Ticket(
    @SerializedName(ID) val id: Long,
    @SerializedName(SUBJECT) @Expose val subject: String,
    @SerializedName(TEXT) @Expose val text: String,
    @SerializedName(EMAIL) @Expose val email: String,
)